tcf_website/tests/test_review.py

```diff
@@ -0,0 +1,77 @@
+# pylint: disable=no-member
+"""Tests for Review model."""
+from urllib.parse import urlencode
+
+from django.contrib.messages import get_messages
+from django.test import TestCase
+from django.urls import reverse
+
+from .test_utils import setup
+
+
+class EditReviewTests(TestCase):
+    """Tests for the `edit_review` view."""
+
+    def setUp(self):
+        setup(self)
+        self.valid_data_with_different_text = urlencode({
+            'text': 'new text',
+            'course': self.review1.course.id,
+            'instructor': self.review1.instructor.id,
+            'semester': self.review1.semester.id,
+            'instructor_rating': self.review1.instructor_rating,
+            'difficulty': self.review1.difficulty,
+            'recommendability': self.review1.recommendability,
+            'enjoyability': self.review1.enjoyability,
+            'amount_reading': self.review1.amount_reading,
+            'amount_writing': self.review1.amount_writing,
+            'amount_group': self.review1.amount_group,
+            'amount_homework': self.review1.amount_homework,
+        })
+
+    def test_nonexistent_review_id(self):
```

Comment:
Seems like this test and the one after it prints unnecessary error messages to console:

![image](https://user-images.githubusercontent.com/55100084/105951852-ab1d6000-603e-11eb-8bbe-d136c509b478.png)

This issue seems similar to what @joek13 is dealing with when testing the delete review functionality (#359) - maybe when one of you guys figures it out you should let the other know.

Reply:
Fixed in 027149ed53a72a0c7ba41eac567c1ab6ddf5f7d1